crates/ironrdp-session/src/fast_path.rs

```diff
@@ -25,6 +25,7 @@ pub enum UpdateKind {
     PointerDefault,
     PointerHidden,
     PointerPosition { x: u16, y: u16 },
+    NativePointerUpdate(Rc<DecodedPointer>),
```

Comment:
Suggestion: I feel "Native" prefix is a bit misleading. My understanding is that this is really a "PointerIcon" (we decoded the icon). Even if this is indeed a "native" cursor, it’s surprising we have "PointerHidden" (is the "Native" part really relevant?). Note also that the "Update" suffix is redundant (the enum is called "UpdateKind").

Reply:
Yeah, let's rename it to `PointerBitmap`.

`PointerHidden` is still needed if the remote wants to hide the pointer completely (e.g. it is the case with 3D-accelerated application which draws the cursor via its game/graphics engine

Reply:
Oh, sorry. I didn’t mean to remove `PointerHidden` of course. I wanted to point out the difference between the two (all the others does not have the "Native" prefix). I think I forgot to write the rest of the sentence :sweat_smile: